src/server/api/utility.ts

```
@@ -0,0 +1,1311 @@
+      election.hasResults ||
+      election.isArchived
+    ) {
+      return errAlias({
+        status: apiErrorStatus.Forbidden,
+        type: apiErrorTypes.ForbiddenActivityMismatch,
+        message: "You cannot edit an active election!!!",
+      });
+    }
+
+    return okAlias<void>().Inconsequential().build();
+  });
```

Biome (1.9.4)

[error] This generator function doesn't contain yield.



(lint/correctness/useYield)